web-app/js/smartR/d3Heatmap.js

```diff
@@ -336,7 +338,7 @@ SmartRHeatmap = (function(){
                     var rowValues = [];
                     for(var i = 0; i < significanceValues.length; i++) {
                         var significanceValue = significanceValues[i];
-                        rowValues.push([i, significanceValue]);
+                        rowValues.push([i, Math.abs(significanceValue)]);
```

Comment:
This is from original code. But to my understanding, the purpose of this value is only for sorting, while the real displayed value is actually another field called d.significance.